src/app/hooks/AuthProvider.tsx

```diff
@@ -68,18 +68,17 @@ export function AuthProvider({ children }: { children: ReactNode }) {
 	const config = useAppConfigContext();
 	const storedToken = getStoredToken();
 	const [egoJwt, setEgoJwt] = useState('');
```

Comment:
You've inheritted this line but...
```suggestion
	const [egoJwt, setEgoJwt] = useState<string | undefined>(undefined);
```

Best to use undefined for missing values.

Reply:
This has a few impacts on other type definitions so I've updated `permissions` like this:
`const permissions = egoJwt ? TokenUtils.getPermissionsFromToken(egoJwt) : [];`